docs/docs/cmd/aad/app/app-role-delete.md

```diff
@@ -0,0 +1,70 @@
+# aad app role delete
```

Comment:
We should add a reference to this file in mkdocs.yml so that it's included in the generated docs

Reply:
Updated the mkdocs.yml file to add the reference